### PR TITLE
Design feedback pt3

### PR DESCRIPTION
**Pull request by conqeror**

### CODE REVIEW

File: src/address-book/AddressBook.tsx [hunk 1]
```diff
@@ -218,7 +221,12 @@ const _AddressBook = ({
                 <span className="address">
                   {address}{' '}
                   <Popover content={t(['addressBook', 'address', 'clickToCopy'])} placement="top">
-                    <CopyOutlined className="clickable" onClick={() => copyToClipboard(address)} />
+                    <span
+                      className="clickable"
```
Comment (qwhex) [inline src/address-book/AddressBook.tsx]: This has no effect.

**Review by qwhex — CHANGES_REQUESTED** [review]
Address book:
Copy button: please, set the proper cursor
The focus style is not really visible for the icons when navigating with the keyboard.

Tx history:
EditableAddress still has the old icons.
I'm not sure what's the plan with the sort-by dropdown (will you put it back with a different style?) - anyways, you forgot to remove the style override (.SortByDropdown).

**Review by qwhex — DISMISSED** [review]
> Address book copy button: please, set the proper cursor
> TxHistory: EditableAddress still has the old icons.

These are still not fixed.

**Comment by conqeror** [comment]
@qwhex 
Made focused IconButtons stand out even more:
image
& all the issues should be fixed by now (used IconButtons in tx list - not sure why I forgot about this :) and used proper cursor for all IconButtons).

Regarding dropdown, it's actually not present in designs at all. Discussed it with Russ and he asked me to just remove it. So I guess it's not worth to override the style for it when we actually don't use it :)

**Review by qwhex — APPROVED** [review]
:sunglasses:

**Comment by qwhex** [comment]
I recommend setting up `data-testid`s to query the buttons.
Tip: by using `debug()` you can see the render output while fixing the tests locally, e.g.:
```ts
const {debug, getAllByTestId} = render(...)
debug()
```

**Pull request merged**